plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -338,6 +339,18 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'xAxisLabelRotation',
+            config: {
+              type: 'TextControl',
```

Comment:
Maybe use `SelectControl` to give users the most common options? (Should have `freeForm` enabled so users can use other values).